taskcluster/ci/split-mono-src/kind.yml

```diff
@@ -66,17 +62,12 @@ task-defaults:
             - -c
               # TODO: move number of chunks elsewhere
             - >-
-                export BIN=$MOZ_FETCHES_DIR &&
-                $VCS_PATH/taskcluster/scripts/pipeline/split-mono-taskcluster.sh
-                {split_chunks}
-                /builds/worker/artifacts
-                {split_length}
+                python3 $VCS_PATH/pipeline/translate/splitter.py
+                --output_dir=/builds/worker/artifacts
+                --num_parts={split_chunks}
+                --compression_cmd=zstdmt
                 $MOZ_FETCHES_DIR/mono.$LOCALE.zst
 
-    fetches:
-        toolchain:
-            - preprocess
```

Comment:
Question: What is this change for?

Reply:
I removed deduplication that was done with preprocess/dedupe tool. We already do this on "merge" step so it was redundant.